### PR TITLE
New UI::InstallationDialog class

### PR DESCRIPTION
**Pull request by ancorgs**
The unit tests are still missing. I'll update the PR tomorrow.

### CODE REVIEW

File: library/general/src/lib/ui/installation_dialog.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    def dialog_title
+      ""
+    end
+
+    # Text to display when the help button is pressed
+    #
+    # @return [String]
+    def help_text
+      ""
+    end
```
Comment (jreidinger) [inline library/general/src/lib/ui/installation_dialog.rb]: are those two optional? if not, then you can use abstract method gem we already depend on

File: library/general/src/lib/ui/installation_dialog.rb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+    # Reimplementation of UI::Dialog#create_dialog for the installer
+    #
+    # It ignores UI::Dialog#dialog_options
+    def create_dialog
+      if dialog_options
+        log.info "This is an InstallationDialog, ignoring #dialog_options"
+      end
+
+      Yast::Wizard.CreateDialog if Yast::Mode.normal # allow manual testing
```
Comment (jreidinger) [inline library/general/src/lib/ui/installation_dialog.rb]: I know it is often used in dialog, but I found it quite hacky. I think better approach is to use `Wizard.IsWizardDialog` and only if it is not wizard, open it ( and of course store that you create it and close it when needed.

File: library/general/src/lib/ui/installation_dialog.rb (hunk 1 above)
Comment (ancorgs) [inline library/general/src/lib/ui/installation_dialog.rb]: I wouldn't do them mandatory. It's fine if you want to have a dialog with empty help or empty headline.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**